docs/source/start-routingspec.rst

```diff
@@ -662,22 +717,200 @@ so consider the name before saving.
++------------------------------+-----------------------------------------------------------------+
+|rs -**d** *integer* mySpec    |debug-level; Default: 0 (0-)                                     |
++------------------------------+-----------------------------------------------------------------+
+|rs -**k** *filename* mySpec   |Optional Lock File                                               |
++------------------------------+-----------------------------------------------------------------+
+|rs -**l** *filename* mySpec   |log-file; Default: routing.log                                   |
++------------------------------+-----------------------------------------------------------------+
+|rs -**m** mySpec              |Do NOT apply Sensor min/max limits; Default: false               |
++------------------------------+-----------------------------------------------------------------+
+|rs -**n** *netlist* mySpec    |Netlist Name                                                     |
++------------------------------+-----------------------------------------------------------------+
+|rs -**o** *filename* mySpec   |Status Output File                                               |
++------------------------------+-----------------------------------------------------------------+
+|rs -**p** *propertyset* mySpec|name=value                                                       |
++------------------------------+-----------------------------------------------------------------+
+|rs -**s** *scriptname* mySpec |ScriptName                                                       |
++------------------------------+-----------------------------------------------------------------+
+|rs -**e** mySpec              |(deprecated -- does nothing)                                     |
++------------------------------+-----------------------------------------------------------------+
+
+Manual Routing Specs Common Options
+-----------------------------------
+
+Troubleshooting - Use Debugger
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+... more content coming soon ...
+
+Backfilling - Use Since and Until
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+... more content coming soon ...
+
+Lockfile Significance
+~~~~~~~~~~~~~~~~~~~~~
+
+... more content coming soon ...
+
+9.Routing Scheduler
+===================
+
+Now that you have set up a routing spec, you may wish to automate it.  
+This is so that a user does not have to manually run the routing spec
+as described above.  Depending on the system where the database is running,
+and where the continuous processes are going to run, the following methods 
+can be used to automate the scheduling of the routing specs.  The first
+method is using an appropriate system job scheduler. This is independent 
+of the OpenDCS software. A few options are outlined below.
 
+System Scheduler
+----------------
+
+For a Windows System the following job schedulers are options:
+* Windows Scheduler
+* Python Crond
```

Comment:
Python is not built into windows.  
do you mean: https://pypi.org/project/python-crontab/ ?